tests/ios/tests/test_ios_profile_options.py

```diff
@@ -69,6 +69,106 @@ def test_validate_video_settings_cell_elements(self, set_capabilities, setup_log
         wifi_cell = global_contents.get_element_by_id(set_capabilities, ios_elements.profile_options_wifi_cell)
         assert wifi_cell.get_attribute('visible') == 'true'
 
+    def test_allow_cellular_download_smoke(self, set_capabilities, setup_logging):
+        """
+        Verify that wifi switch is turned ON by default
+        Verify that turning the switch ON again will open Allow cellular download popup
+        Verify that cellular download popup show these elements
+        "Allow Cellular Download" as Title
+        Allow cellular download Description
+        Don't allow button
+        Allow button
```

Comment:
can we add indents to these with dashes for e.g. 
```suggestion
        - "Allow Cellular Download" as Title
        - Allow cellular download Description
        - Don't allow button
        - Allow button
```

Reply:
i have added indentations but not added the dashes, because if i add dashes then it will be different from the whole code.